HISTORY.rst

```diff
@@ -6,10 +6,25 @@ History
 .. to_doc
 
 ---------------------
-0.9.1.dev0
----------------------
-
-    
+0.10.0.dev0
+---------------------
+
+* Refactor shed tool functionality to be more modular, readable, and concise
```

Comment:
Maybe rephrase this to the things that matter to the end user:

Removed deprecated options from shed-tools CLI. shed-tools update now also accepts tool list, so tools in galaxy can be selectively updated. 
Improved algorithm leads to much faster skipping of already installed tool, which makes the installation of tools much faster on an already populated galaxy.

Reply:
@rhpvorderman I am sure @jmchilton does not mind if you re-phrase this to your liking with a direct commit. Please do so.

Reply:
@martenson Thanks for the suggestion. I changed it.